app/controllers/stations_controller.rb

```diff
@@ -6,5 +6,6 @@ def index
 
   def show
     @station = Station.find_by(slug: params[:slug])
+    @show_data = @station.show_data unless (@station.start_trips && @station.end_trips)
```

Comment:
the alternative here is to put something like this in the view:
```ruby
<% unless @station.start_trips && @station.end_trips %>
  do whatever you need to show the data
<% else %>
  Sorry, this station has no trips associated with it. Nothing to see here!
<% end %>
```

Reply:
otherwise you STILL need to check in the view that `@show_data` even has anything in it too